src/Domain/DonationService.php

```diff
@@ -74,12 +70,6 @@ class DonationService
         'The parameter application_fee_amount cannot be updated on a PaymentIntent after a capture has already been made.',
     ];
 
-
-    // for now we are opting out of async - when we go to async capture we will need to listen to the
-    // webhooks to update donations and also regular giving mandates. See
-    // https://docs.stripe.com/payments/payment-intents/asynchronous-capture#listen-webhooks
-    // and MAT-395
```

Comment:
I just moved over the issue ref part of this comment, as I'm not convinced we need the extra stuff Stripe are talking about getting from webhooks there.